openmdao/docs/_utils/create_feature_docs_unit_test_db.py

```diff
@@ -31,166 +31,164 @@
-    out = []
-    out += self._str_signature()
-    out += self._str_summary()
-    out += self._str_extended_summary()
-    for param_list in ('Parameters', 'Returns', 'Yields',
-                       'Other Parameters', 'Raises', 'Warns'):
-        out += self._str_param_list(param_list)
-    out += self._str_section('Warnings')
-    out += self._str_see_also(func_role)
-    for s in ('Notes', 'References', 'Examples', 'Features'):
-        out += self._str_section(s)
-    for param_list in ('Attributes', 'Methods'):
-        out += self._str_param_list(param_list)
-    out += self._str_index()
-    return '\n'.join(out)
-
-
-# Do the actual patch switchover to these local versions
-NumpyDocString.__init__ = __init_NumpyDocString__
-NumpyDocString.__str__ = __str_NumpyDocString__
-SphinxDocString.__str__ = __str_SphinxDocString__
-# --------------end monkeypatch---------------------
-
-
-# remove any existing sqlite database file
-try:
-    os.remove(sqlite_file)
-except OSError:
-    pass
-
-# Prepare to write to the sqlite database file
-conn = sqlite3.connect(sqlite_file)
-cur = conn.cursor()
-cur.execute("CREATE TABLE {tn}(method_path TEXT, feature TEXT, title TEXT, \
-                unit_test_source TEXT, run_outputs TEXT)".format(tn=table_name))
-
-# Search for all the unit tests
-suites = []
-for listing in os.listdir(".."):
-    if os.path.isdir(os.path.join("..", listing)):
-        if listing not in IGNORE_LIST:
-            suites.append(unittest.TestLoader().discover(os.path.join('..', listing), pattern="test_*.py"))
-
-for suite in suites:
-    for test_file in suite:  # Loop through the TestCases found
-        for test_case in test_file:  # Loop though the test methods in the TestCases
-            if isinstance(test_case, unittest.suite.TestSuite):
-                for test_method in test_case:
-                    cls = test_method.__class__
-                    module_path = cls.__module__
-                    class_name = cls.__name__
-                    method_name = test_method._testMethodName
-                    method_path = '.'.join([module_path, class_name, method_name])
-                    test_doc = getattr(cls, method_name).__doc__
-                    if test_doc:
-                        test_doc_numpy = NumpyDocString(test_doc)
-                        if test_doc_numpy['Features']:
-                            for feature in [f.strip() for f in test_doc_numpy['Features']]:
-                                title = test_doc_numpy['Summary'][0]
-                                unit_test_source, run_outputs, skipped, failed = \
-                                    get_unit_test_source_and_run_outputs(method_path)
-                                cur.execute(
-                                    'insert into {tn} values (?,?,?,?,?)'.format(tn=table_name),
-                                    (method_path, feature, title, unit_test_source, run_outputs))
-
-conn.commit()
-conn.close()
+#import importlib
```

Comment:
just delete the file?  it's always there in the repo if it's needed again...

Reply:
Ok, I will delete that old stuff we don't use.